__tests__/e2e tests/auth.test.js

```
@@ -0,0 +1,172 @@
+    phone: "1234567890",
+    dateOfBirth: "01-01-2000",
+  };
+
+  afterEach(() => {
+    vitest.restoreAllMocks();
+  });
+
+  describe("General Validation Errors", () => {
+    test("should return 400 if name is missing.", async () => {
+      const body = { ...mockCustomer };
+      delete body.name;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)